ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -139,48 +144,62 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
 
     @objc public weak var delegate: TwoLineTitleViewDelegate?
 
-    private var alignment: Alignment = .center
+    var currentStyle: Style {
+        didSet {
+            applyStyle()
+        }
+    }
+
+    private lazy var alignmentConstraint: NSLayoutConstraint = centerXAnchor.constraint(equalTo: containingStackView.centerXAnchor)
+    private var alignment: Alignment = .center {
+        didSet {
+            guard alignment != oldValue else {
+                return
+            }
+            alignmentConstraint.isActive = false
+            let keyPath = alignment.xAxisKeyPath
+            alignmentConstraint = self[keyPath: keyPath].constraint(equalTo: containingStackView[keyPath: keyPath])
+            alignmentConstraint.isActive = true
+        }
+    }
     private var interactivePart: InteractivePart = .none
     private var animatesWhenPressed: Bool = true
     private var accessoryType: AccessoryType = .none
 
-    private let titleButton = EasyTapButton()
-    private var titleAccessoryType: AccessoryType {
-        return interactivePart.contains(.title) ? accessoryType : .none
-    }
+    private lazy var containingStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 0
+        return stackView
+    }()
 
-    var currentStyle: Style {
-        didSet {
-            applyStyle()
-        }
+    private static func makeEmptyTitleLineStackView() -> UIStackView {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.spacing = TokenSetType.titleStackSpacing
+        return stackView
     }
 
-    private lazy var titleButtonLabel: Label = {
+    private lazy var titleContainer = Self.makeEmptyTitleLineStackView()
+    private lazy var subtitleContainer = Self.makeEmptyTitleLineStackView()
```

Comment:
Do these need to be lazy? If they're not referencing anything from an initialized instance, they should be fine just being `var`s. Especially if they're just referencing a static func.

Reply:
Turns out that initializing these in `init` saves on binary size.